sphinx_gallery/gen_rst.py

```diff
@@ -447,12 +447,14 @@ def execute_script(code_block, example_globals, image_path, fig_count,
                 image_list += HLIST_IMAGE_TEMPLATE % figure_name.lstrip('/')
 
     except Exception:
-        figure_list = []
-        image_list = '%s is not compiling:' % src_file
         print(80 * '_')
-        print(image_list)
+        print('%s is not compiling:' % src_file)
         traceback.print_exc()
         print(80 * '_')
+
+        figure_list = []
+        exception_msg = indent(traceback.format_exc(), '    ')
```

Comment:
Nitpick: in order not to do traceback.print_exc() and then traceback.format_exc() which is kind of doing the same thing twice, you can do:

```python

formatted_exception = traceback.format_exc()

print(formatted_exception)

exception_msg = indent(formatted_exception, ' ' * 4)
```